docs-md/developer-guide/ksqldb-reference/print.md

```diff
@@ -40,7 +36,7 @@ The following statement shows how to print all of the records in a topic named
 `ksql__commands`.
 
 ```sql
-PRINT 'ksql__commands' FROM BEGINNING;
+PRINT ksql__commands FROM BEGINNING;

```

Comment:
note: the PRINT command is now case-sensitive. (As it being case-insensitive was causing poor UX).